Content.Client/Xenobiology/XenobioTubeVisualiser.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using JetBrains.Annotations;
+using Robust.Client.GameObjects;
+using Content.Shared.Xenobiology;
+using static Content.Shared.Xenobiology.SharedXenoTubeComponent;
+
+namespace Content.Client.Cloning
+{
+    [UsedImplicitly]
+    public class XenobioTubeVisualiser : AppearanceVisualizer
+    {
+        public override void OnChangeData(AppearanceComponent component)
+        {
+            base.OnChangeData(component);
+
+            var sprite = component.Owner.GetComponent<ISpriteComponent>();
+            if (!component.TryGetData(XenoTubeStatus.Powered, out bool powered)) ;// return;
+            if (!component.TryGetData(XenoTubeStatus.Occupied, out bool occupied)) ;// return;
+            if (!powered) sprite.LayerSetState(0, "tube-unpowered");
+            else if (powered)
+            {
+                if (!occupied) sprite.LayerSetState(0, "tube-powered");
+                else if (occupied) sprite.LayerSetState(0, "tube-occupied");
+            }
```

Comment:
What is wrong?

Reply:
Not sure what is the issue here

Reply:
![image](https://user-images.githubusercontent.com/16618384/130439090-8200b442-22d1-43d6-8c9a-e41f01b8d62d.png)
I don't like just shitposting so i will break down everything bad with this for you:
- commented out return statements after the first two ifs, making the ifs literally do nothing. The TryGet Pattern exists so that you either return if the tryget fails, or nest the usage of the out parameters inside the if. You are doing neither.
- third if is first without brackets, then with brackets. please use it consistently
- `if(!occupied) else if (occupied)` and `if(!powered) else if (powered)` is literally just `if(!occupied) else` with extra steps
- i also dislike that you are generally just using the bracket-less version of the ifs, i think it looks messy, but thats just personal preference